src/openforms/authentication/contrib/digid_eherkenning_oidc/views.py

```diff
@@ -84,6 +99,18 @@ class DigiDOIDCAuthenticationCallbackView(
     plugin_identifier = "digid_oidc"
     auth_backend_class = OIDCAuthenticationDigiDBackend
 
+    def get_error_message_parameters(
+        self, error: str, error_description: str
+    ) -> tuple[str, str]:
+        if (
+            error == "access_denied"
+            and error_description
+            and error_description == "The user cancelled"
+        ):
+            return ("_digid-message", LOGIN_CANCELLED)
```

Comment:
```suggestion
            return (DIGID_MESSAGE_PARAMETER, LOGIN_CANCELLED)
```
re-use the constant?